gcc/rust/ast/rust-pattern.h

```diff
@@ -191,6 +191,20 @@ class WildcardPattern : public Pattern
   }
 };
 
+class RestPattern : public Pattern
+{
+  Location locus;
+
+public:
+  std::string as_string () const override { return std::string (".."); }
```

Comment:
```suggestion
  std::string as_string () const override { return ".."; }
```
Also, small nitpick, it would be better to have the implementation in `rust-ast.cc`. But it's a bit overkill for such a small function so feel free to ignore :)